DQM/EcalMonitorTasks/interface/TimingTask.h

```diff
@@ -19,10 +19,18 @@ namespace ecaldqm {
     void runOnRecHits(EcalRecHitCollection const&, Collections);
     void runOnUncalibRecHits(EcalUncalibratedRecHitCollection const&);
 
+    enum Constants {
+      nBXBins = 15
+    };
+
   private:
     void beginLuminosityBlock(edm::LuminosityBlock const&, edm::EventSetup const&) override;
+    void beginEvent(edm::Event const&, edm::EventSetup const&) override;
     void setParams(edm::ParameterSet const&) override;
 
+    int bxBinEdges_[nBXBins + 1];
```

Comment:
Hi @davidlange6 , thanks very much for that suggestion. It does indeed improve code readability. I've tested the changes and added a commit to this PR and to the 900 PR: https://github.com/cms-sw/cmssw/pull/18069

Please let me know if you have other suggestions. Thanks!